### PR TITLE
docs: blog post about graphback

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: website/blog/2019-08-21-introduction.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Graphback generates Node.JS application with an entire codebase that is editable and can be later deployed to your own server or cloud provider.
+By default, Graphback will follow the best patterns and include proven open-source libraries provided by the vibrant GraphQL community. When working with relational databases Graphback will allow users to manage database schema. The real power of Graphback is its code generation engine that can be extended for different needs. 
+
+![diagram](https://thepracticaldev.s3.amazonaws.com/i/c9q6rvbb9v534073cq6j.png)
+
+
+## Graphback to resolve major issues in GraphQL community
+
+Graphback helps developers to boost their getting started the experience with GraphQL. Graphback CLI offers different workflows supporting fetching data from the database and rest endpoints.
```
Comment (MStokluska) [inline website/blog/2019-08-21-introduction.md]: REST not rest?

Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Graphback helps developers to boost their getting started experience with GraphQL. The Graphback CLI offers different workflows supporting data retrieval from the database and REST endpoints.
```

File: website/blog/2019-08-21-introduction.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Graphback generates Node.JS application with an entire codebase that is editable and can be later deployed to your own server or cloud provider.
+By default, Graphback will follow the best patterns and include proven open-source libraries provided by the vibrant GraphQL community. When working with relational databases Graphback will allow users to manage database schema. The real power of Graphback is its code generation engine that can be extended for different needs. 
+
+![diagram](https://thepracticaldev.s3.amazonaws.com/i/c9q6rvbb9v534073cq6j.png)
+
+
+## Graphback to resolve major issues in GraphQL community
+
+Graphback helps developers to boost their getting started the experience with GraphQL. Graphback CLI offers different workflows supporting fetching data from the database and rest endpoints. 
+Templates allow having different flavors of the GraphQL frameworks like Relay, Apollo, and Prisma for database access. Thanks to templates developers can avail of a fully functional set of the libraries that produce business value and work really well together. The community can provide their own templates to contribute to the success of the GraphQL entire ecosystem.
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Templates allow having different flavors of the GraphQL frameworks like Relay, Apollo, and Prisma for database access. Thanks to templates developers can avail of a fully functional set of libraries that produce business value and work really well together. The community can provide their own templates to contribute to the success of the entire GraphQL ecosystem.
```

Comment (craicoverflow) [reply website/blog/2019-08-21-introduction.md]: Should "flavors" be "flavours"? It depends on if we want US English or Irish/British.

File: website/blog/2019-08-21-introduction.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+## Graphback to resolve major issues in GraphQL community
+
+Graphback helps developers to boost their getting started the experience with GraphQL. Graphback CLI offers different workflows supporting fetching data from the database and rest endpoints. 
+Templates allow having different flavors of the GraphQL frameworks like Relay, Apollo, and Prisma for database access. Thanks to templates developers can avail of a fully functional set of the libraries that produce business value and work really well together. The community can provide their own templates to contribute to the success of the GraphQL entire ecosystem. 
+
+Graphback generated application source code will be always in your full control - no hidden dependencies or any additional limitations included from Graphback runtime. Graphback will generate CRUD data access methods that can be extended into more customized data access functions at any time.
+
+Developers can introduce custom queries and mutations that will generate data access stubs. Thanks to first-class TypeScript support generated projects offer type safety.
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Developers can introduce custom queries and mutations that will generate data access stubs. Thanks to first-class TypeScript support, generated projects offer type safety.
```

File: website/blog/2019-08-21-introduction.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Templates allow having different flavors of the GraphQL frameworks like Relay, Apollo, and Prisma for database access. Thanks to templates developers can avail of a fully functional set of the libraries that produce business value and work really well together. The community can provide their own templates to contribute to the success of the GraphQL entire ecosystem. 
+
+Graphback generated application source code will be always in your full control - no hidden dependencies or any additional limitations included from Graphback runtime. Graphback will generate CRUD data access methods that can be extended into more customized data access functions at any time.
+
+Developers can introduce custom queries and mutations that will generate data access stubs. Thanks to first-class TypeScript support generated projects offer type safety.  
+
+## Ecosystem of templates
+
+Graphback builds an ecosystem for developers that can build sample apps as templates that can be extended by additional features and business logic. Graphback templates follow canonical implementations and allow developers to continue development traditional way and tweak every aspect of the application.
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Graphback builds an ecosystem for developers that can build sample apps as templates that can be extended by additional features and business logic. Graphback templates follow canonical implementations and allow developers to continue development in a traditional way and tweak every aspect of the application.
```

File: website/blog/2019-08-21-introduction.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+Developers can introduce custom queries and mutations that will generate data access stubs. Thanks to first-class TypeScript support generated projects offer type safety.  
+
+## Ecosystem of templates
+
+Graphback builds an ecosystem for developers that can build sample apps as templates that can be extended by additional features and business logic. Graphback templates follow canonical implementations and allow developers to continue development traditional way and tweak every aspect of the application. 
+
+![Alt Text](https://thepracticaldev.s3.amazonaws.com/i/ss0zueqh885m8h5wra1y.gif)
+
+## Not only GraphQL Server
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
## Not just a GraphQL server
```

File: website/blog/2019-08-21-introduction.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+Apart from fully functional backend server developers can choose to generate GraphQL queries for the client-side applications. Client-side code can utilize form generators like https://uniforms.tools giving developers end to end front-end and back-end solutions that will rely on their schema.
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Apart from fully functional back-end server, developers can choose to generate GraphQL queries for client-side applications. Client-side code can utilize form generators like https://uniforms.tools, giving developers end to end front-end and back-end solutions that will rely on their schema. 
```

File: website/blog/2019-08-21-introduction.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+![Alt Text](https://thepracticaldev.s3.amazonaws.com/i/ss0zueqh885m8h5wra1y.gif)
+
+## Not only GraphQL Server
+
+Apart from fully functional backend server developers can choose to generate GraphQL queries for the client-side applications. Client-side code can utilize form generators like https://uniforms.tools giving developers end to end front-end and back-end solutions that will rely on their schema. 
+
+## Support 
+
+Our project is maintained by volunteers who are devoted to constant maintenance.
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Our project is maintained by volunteers who are devoted to constant love and maintenance. 
```

File: website/blog/2019-08-21-introduction.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+![Alt Text](https://thepracticaldev.s3.amazonaws.com/i/ss0zueqh885m8h5wra1y.gif)
+
+## Not only GraphQL Server
+
+Apart from fully functional backend server developers can choose to generate GraphQL queries for the client-side applications. Client-side code can utilize form generators like https://uniforms.tools giving developers end to end front-end and back-end solutions that will rely on their schema. 
+
+## Support 
+
+Our project is maintained by volunteers who are devoted to constant maintenance. 
+You can support us by staring project on GitHub or posting on social media
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
You can support us by becoming a [Graphback stargazer](https://github.com/aerogear/graphback/stargazers) on GitHub, or by sharing it on social media.
```

File: website/blog/2019-08-21-introduction.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+
+Our github:
```
Comment (craicoverflow) [inline website/blog/2019-08-21-introduction.md]:
```suggestion
Our GitHub: 
```

**Pull request merged**